kolibri/core/assets/src/navigation.vue

```diff
@@ -12,16 +14,15 @@
 
 
 <script>
+  /* global _navItems window*/
 
   module.exports = {
     data() {
       return {
         // items that go in the title bar
-        nav_items: window._nav ? window._nav.nav_items || [] : [],
-        title_bar: {
-          title: 'Kolibri',
-          home_link: '/',
-        },
+        nav_items: _navItems || [],
+        // the current app on
+        app: window.location.pathname.split('/')[1],
```

Comment:
@rtibbles figured this'll work for now. Do you think it's acceptable for long term use?